test/java/io/envoyproxy/envoymobile/engine/testing/QuicTestServer.java

```diff
@@ -0,0 +1,59 @@
+package io.envoyproxy.envoymobile.engine.testing;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Wrapper class to start a Quic test server.
+ */
+public final class QuicTestServer {
+
+  private static final String CERT_USED = "upstreamcert.pem";
+  private static final String KEY_USED = "upstreamkey.pem";
+
+  private static final AtomicBoolean sServerRunning = new AtomicBoolean();
+
+  /*
+   * Starts the server.
```

Comment:
Comment what happens if the server is already running?

Reply:
Done